node_modules/oae-core/lhnavigation/lhnavigation.html

```diff
@@ -5,7 +5,7 @@
     <div class="oae-lhnavigation">
         <ul class="nav nav-list"><!-- --></ul>
     </div>
-    <div class="oae-page"><!-- --></div>
+    <div id="oae-page-content" class="oae-page"><!-- --></div>
```

Comment:
Can you confirm that jumping to the actual content of the page is the intended behavior? The previous behavior just made it jump until right after the top navigation (but before the left hand navigation).